common/web/types/src/util/compiler.ts

```diff
@@ -0,0 +1,17 @@
+/**
```

Comment:
Can we rename this to CompilerInterfaces.ts? Clarifies that it isn't the compiler itself.